src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -722,93 +782,107 @@ uct_gdaki_md_check_uar(uct_ib_mlx5_md_t *md, CUdevice cuda_dev)
     return status;
 }
 
+static int uct_gdaki_is_peermem_loaded(const uct_ib_md_t *md)
```

Comment:
we can probably remove peermem_loaded cached variable and this wrapper altogether, and take it directly from md->reg_mem_types